awesomerc.lua

```diff
@@ -561,3 +566,4 @@ end)
 client.connect_signal("mouse::enter", function(c)
     c:activate { context = "mouse_enter", raise = false }
 end)
+
```

Comment:
and here

Reply:
A blank line. Okay, done.

The good news is that I only added test code to `awesomerc.lua`. The other files shouldn't have test code on them.

If this time everything goes okay, I assume you'll want me to squash all the commits between 392eff73f001d163ec6649fc90657eb6fefdac89 and 934052d2381398f21aff7f43dc6f65a97eedd651 into a single one?